cmd/gh-slack/cmd/auth.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/cli/go-gh/pkg/config"
+	"github.com/rneatherway/slack"
+	"github.com/spf13/cobra"
+)
+
+var authCmd = &cobra.Command{
+	Use:   "auth [flags]",
+	Short: "Prints authentication information for the Slack API",
+	Long:  `Prints authentication information for the Slack API.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cfg, err := config.Read()
+		if err != nil {
+			return err
+		}
+
+		team, err := getFlagOrElseConfig(cfg, cmd.Flags(), "team")
+		if err != nil {
+			return err
+		}
+
+		auth, err := slack.GetCookieAuth(team)
+		if err != nil {
+			return err
+		}
+
+		vals := url.Values{}
+		for k, v := range auth.Cookies {
+			vals.Add(k, v)
+		}
+
+		fmt.Printf("export SLACK_TOKEN=%s\n", auth.Token)
+		fmt.Printf("export SLACK_COOKIES=%s\n", vals.Encode())
+		return nil
+	},
+	Example: `  gh-slack auth [-t <team-name>]
+	` + configExample,
+}
+
+func init() {
+	authCmd.Flags().StringP("team", "t", "", "Slack team name (required here or in config)")
+	authCmd.SetUsageTemplate(authCmdUsage)
+	authCmd.SetHelpTemplate(authCmdUsage)
+}
+
+const authCmdUsage string = `Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}}{{end}}{{if gt (len .Aliases) 0}}
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
```

Comment:
But I'd like to have a longer message explaining the criticality of this in the description as well. Maybe this?

```suggestion

Security:
  Treat those values as secrets and don't share them with anyone.
  If someone get access to them, they will be able to impersonate you.

```

Reply:
I also am wondering if there's a way to _reset_ the cookie so that we can add a sectin saying

```
In case of leak, run `slack auth renew`
```